### PR TITLE
Add subctrl mapping

### PR DESCRIPTION
**Pull request by minhaj10p**

### CODE REVIEW

File: generator/mapper.go [hunk 1]
```diff
@@ -74,62 +75,84 @@ func CreateCatalogsFromProfile(profileArg *profile.Profile) ([]*catalog.Catalog,
 		}
 	}
 }
+func getSubControl(call profile.Call, ctrls []catalog.Control, helper impl.Catalog) (*catalog.Subcontrol, error) {
```
Comment (anweiss) [inline generator/mapper.go]: @minhaj10p why the need to return a pointer to a subcontrol here?

Comment (minhaj10p) [reply generator/mapper.go]: just so I could return a nil if not found. nothing special. should I remove the pointer?

Comment (anweiss) [reply generator/mapper.go]: Ah gotcha. Yea in this case it's probably fine just to return an empty struct vs. having to dereference the pointer every time in the rest of your code.

Comment (minhaj10p) [reply generator/mapper.go]: Changes pushed @anweiss

**Pull request merged**